Translation: chapter 6 (#83)
btcven/locha: documentation/en/chapter_6.md

# 6. Locha Mesh Hardware 
 

> This section presents the parameters that were used to choose all the control and transmission devices currently used in Locha Mesh. 

Reviewer: ```suggestion
This section aims to show parameters that were used to choose all control and transmission devices currently used in Locha Mesh. 
```

## 6.1 Radio frequency module 

<img src="imple_pic/RF-selection.svg"  height="650" width="450" align="left"/>

> To choose the radio module that contains the microcontroller (MCU), the RF and modulation needed to transmit on the 915 MHz frequency, some parameters were established:

Reviewer: ```suggestion
To choose the radio module that contains the microcontroller (MCU), the RF and modulation needed to transmit on  915 MHz frequency, some parameters were established:
```

<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>

## 6.4 Battery power control 
<img src="imple_pic/BatteryManagementSystem-selection.svg"  height="450" width="550" align="left"/>

The power control is the system that is responsible for reading variables related to the battery such as available power, charging time, current supplied, etc. 

When selecting the device that will perform this task, it must have I2C support, in the chart you can see two different devices from two different manufacturers, which have similar presentations but differ greatly in cost per unit, so The BQ27441-G1 will be used as a device for collecting battery related data to be sent to the main control unit. 

<br>
<br>

## 6.5 MCU (Interface)

For the selection of the microcontroller, we have focused on the  _Espressif_ controllers, as this is a brand that offers a variety of devices with incredible features such as Bluetooth and Wi-Fi at very low cost, which another brand could hardly offer in addition to the extensive documentation that exist for this device. 

Benefits of the ESP32 family:

> - Economical.

Author: ```suggestion
- Low-cost.
```

- Extensive documentation available.
- Examples on the Arduino platform that can be easily carried into the espressif environment. 

> - Energy- efficient.

Author: ```suggestion
- Energy efficient.
```
